source/reference/numeric.rst

```diff
@@ -49,36 +49,11 @@ The ``integer`` type is used for integral numbers.
 
 .. warning::
 
-    The precise treatment of the "integer" type may depend on the
-    implementation of your JSON Schema validator.  JavaScript (and
-    thus also JSON) does not have distinct types for integers and
-    floating-point values.  Therefore, JSON Schema can not use type
-    alone to distinguish between integers and non-integers.  The JSON
-    Schema specification recommends, but does not require, that
-    validators use the mathematical value to determine whether a
-    number is an integer, and not the type alone.  Therefore, there is
-    some disagreement between validators on this point.  For example,
-    a JavaScript-based validator may accept ``1.0`` as an integer,
-    whereas the Python-based `jsonschema
-    <https://pypi.python.org/pypi/jsonschema>`__ does not.
-
-Clever use of the ``multipleOf`` keyword (see `multiples`) can be used
-to get around this discrepancy.  For example, the following likely has
-the same behavior on all JSON Schema implementations:
-
-.. schema_example::
-
-    { "type": "number", "multipleOf": 1.0 }
-    --
-    42
-    --
-    42.0
-    --X
-    3.14156926
-
-
-.. index::
-   single: number
+    JSON does not have distinct types for integers and floating-point
+    values. Therefore, implementations can not use the type of a value
+    alone to distinguish between integers and non-integers. Validators
+    should use the mathematical value to determine whether a number is
+    an integer or a number.
```

Comment:
I think it's still worth mentioning an example. I was head scratching for a moment.

The fact that 1.0 is mathmatically equivilent to 1 and is therefore for the purposes of JSON Schema considered an Integer as opposed to a non-integer is worth calling out explicitly as an example IMHO.

The behaviour for previous drafts was iirc ambigious in this respect, hence implementations not agreeing.

Reply:
I ended up moving this out of a warning and just putting in it in the main description of an integer. I also added `1.0` to the examples.